### DISCUSSION TITLE
Is it possible to export all custom translations and import them into another Kimai installation?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#4667] [Questions & Answers] [answered]
Can we export custom translations?

**Comment by kevinpapst** [comment, marked as answer]
You can dump and re-import the database table `kimai2_translations`.

Please post questions regarding paid plugins next time at: https://github.com/kevinpapst/kimai-plugins

**Reply by EduNetEurope** [reply]
Thank you, Kevin!
